NitroxClient/MonoBehaviours/MultiplayerSettings.cs

```diff
@@ -0,0 +1,77 @@
+﻿using NitroxClient.MonoBehaviours.Gui.Settings;
+using NitroxClient.Unity.Helper;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace NitroxClient.MonoBehaviours
+{
+    public class MultiplayerSettings : MonoBehaviour
+    {
+        private int TabIndex;
+        private uGUI_OptionsPanel OptionsPanel;
+
+        private void Start()
+        {
+            OptionsPanel = gameObject.RequireComponent<uGUI_OptionsPanel>();
+
+            TabIndex = OptionsPanel.AddTab("Multiplayer");
+            AddInputFieldOption("Name", "Enter Name", SettingsManager.GetName(), new UnityAction<string>(newName => SettingsManager.Name= newName));
```

Comment:
For some reason the UI has a different scale when in game, compared to the main menu. This causes the textbox to be partially cut off...
![image](https://user-images.githubusercontent.com/2325264/32075502-4fec055a-ba9d-11e7-947a-fda8fccb7d31.png)